content/home.md

```diff
@@ -0,0 +1,42 @@
+type: page
+slug: home
+body_class: home
+title: Home
+---
+
+PyCon UK is the annual gathering of the UK Python community and its friends
+from around the world.  This year, the conference is held at [Cardiff City
+Hall](http://www.cardiffcityhall.com/), from Thursday 15th to Monday 19th
+September.
+
+We&rsquo;ll be putting on a programme of talks, workshops, and other events
+aimed at the whole Python community, from web developers to hardware engineers,
+from professionals to hobbyists.  There will be an education programme for
+teachers, and an expanded research programme aimed at scientists and
+researchers.
```

Comment:
can we change "aimed at scientists and researchers" to "aimed at researchers from the humanities, arts and sciences." please.